find_metrics.py

```diff
@@ -122,6 +148,10 @@ def find_search_windows(genes):
     
 def find_overlaps(elements, genes):
     
+    #PyRanges is used to find specified element type overlaps within search
```

Comment:
What are you finding the overlaps of? Make the name of this function a little longer; your function below this one is a good example of what would be better

Reply:
Changed function name